lib/dartino/dartino_util.dart

```diff
@@ -137,13 +137,12 @@ class _Dartino {
       var sdk = sdkFor(null);
       if (sdk != null && sdk.validate()) {
 
-        // Given a valid Dartino SDK, now validate the Dart SDK
-        if (sdkManager.sdk?.isValidSdk != true) {
+        // If there's no Dart SDK configured, use the one provided by Dartino.
+        if (sdkManager.noSdkPathConfigured) {
```

Comment:
If the Dart SDK is not valid (e.g. path is set but directory has been removed) they wouldn't you want the sdkPath set?